src/config.h

```diff
@@ -0,0 +1,10 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+extern struct Config_t
+{
+    //! Record gameplay data
+    bool    RecordGameplayData = false;
+} g_config; // main_config.cpp
```

Comment:
A good structure, I will need to move here all other global settings were toggled by settings/thextech.ini, should be more organized! :fox_face: :+1:

Reply:
Glad you like this! :)

I am also thinking in the long term we may be able to combine the compat and config data but that is not something to do now.